### PR TITLE
import export nusmod

### PR DESCRIPTION
**Pull request by CheahHaoYi**
To export and import nusmod to YAMOM

Export command works 
Encounter Exceptions when importing, get invalid module error

### CODE REVIEW

File: src/main/java/seedu/duke/parser/Parser.java [hunk 1]
```diff
@@ -1,42 +1,54 @@
...
-import seedu.duke.command.DeleteModuleCommand;
-import seedu.duke.command.ExitCommand;
-import seedu.duke.command.HelpCommand;
-import seedu.duke.command.IncompleteCommand;
-import seedu.duke.command.InvalidModuleCommand;
-import seedu.duke.command.UnknownCommand;
-import seedu.duke.command.ViewTimetableCommand;
-import seedu.duke.command.SelectSlotCommand;
-import seedu.duke.command.SearchModuleCommand;
+import seedu.duke.command.*;
```
Comment (amitrahman1026) [inline src/main/java/seedu/duke/parser/Parser.java:3]: i think we should avoid wildcard imports

**Review by amitrahman1026 — CHANGES_REQUESTED** [review]
LGTM except for coding style,

can also add to UG & DG in the same pr

**Review by kohnh — COMMENTED** [review]
Minor tweaks

**Review by kohnh — APPROVED** [review]
LGTM

File: src/main/java/seedu/duke/parser/Parser.java (hunk 1 above)
Comment (kohnh) [reply src/main/java/seedu/duke/parser/Parser.java]: Fixed

**Comment by CheahHaoYi** [comment]
Fix #78

**Pull request merged**